pkg/local_workflows/auth_workflow_test.go

```diff
@@ -103,12 +103,6 @@ func Test_auth_token(t *testing.T) {
 }
 
 func Test_pat(t *testing.T) {
-	const (
```

Comment:
Question: why was this and the other lines here removed? might be for a good reason, it is just not clear to me.